src/components/dashboard/news-item.vue

```diff
@@ -0,0 +1,51 @@
+<script>
+import TimeFormat from '~/mixins/time-format'
+export default {
+  name: 'news-item',
+  mixins: [TimeFormat],
+  components: {
+    Chips: () => import('~/components/common/chips.vue')
+  },
+  props: {
+    // news: () => {} // of { title, date, description, author }
+    news: Object
```

Comment:
I meant that this component should accept the values mentioned in the comments as props, instead of the single `news` object - it will just simplify the component and make this explanatory comment unnecessary